src/ol_openedx_chat/static/js/ai_chat.js

```diff
@@ -0,0 +1,75 @@
+function AiChatAsideView(runtime, element) {
+    $('.chat-button').on('click', function () {
+        const blockKey = $(this).data("block-key")
```

Comment:
Minor question... I see the `data-` attribute can be used to pass serialized data from python to the JS file. 

I noticed in https://docs.openedx.org/_/downloads/xblock/en/latest/pdf/#page=60 the JS function had signature
```js
function ThumbsAside(runtime, element, block_element, init_args) { /* etc */ }
```
Do we control the 3rd argument? Is that another way to pass data?